schemas/Activity

```diff
@@ -151,90 +139,59 @@
         {
             "@id": "reproterms:shuffle",
             "@type": "rdf:Property",
-            "rdfs:comment": "An element to determine if the list of items is shuffled or in order.",
+            "rdfs:comment": "An element (bool) to determine if the list of items is shuffled or in order.",
             "rdfs:label": "shuffle",
-            "rdfs:subPropertyOf": {
-                "@id": "reproterms:ui"
-            },
-            "schema:domainIncludes": {
-                "@id": "reproschema:Activity"
-            },
-            "schema:rangeIncludes": {
-                "@id": "schema:boolean"
-            },
-            "schema:sameAs": {
-                "@id": "reproterms:shuffle"
-            }
-        },
-        {
-            "@id": "reproterms:variableMap",
-            "@type": "rdf:Property",
-            "rdfs:comment": "An element to define mapping between variables used and
-             the corresponding nodes.",
-            "rdfs:label": "variableMap",
             "schema:domainIncludes": [
                 {
                     "@id": "reproschema:Protocol"
                 },
                 {
                     "@id": "reproschema:Activity"
                 }
-
             ],
-            "schema:rangeIncludes": [
-                {
-                    "@id": "reproschema:Activity"
-                },
-                {
-                    "@id": "reproschema:Field"
-                }
-            ]
+            "schema:rangeIncludes": {
+                "@id": "schema:boolean"
+            },
+            "schema:sameAs": {
+                "@id": "reproterms:shuffle"
+            }
         },
         {
-            "@id": "reproterms:visibility",
+            "@id": "reproschema:addProperties",
             "@type": "rdf:Property",
-            "rdfs:comment": "An element to describe visibility conditions of items in an assessment.",
-            "rdfs:label": "visibility",
+            "rdfs:comment": "An array of objects to describe the various properties added to assessments and fields.",
+            "rdfs:label": "addProperties",
             "schema:domainIncludes": [
                 {
                     "@id": "reproschema:Protocol"
                 },
                 {
                     "@id": "reproschema:Activity"
                 }
-
             ],
-            "schema:rangeIncludes": [
-                {
-                    "@id": "reproschema:Activity"
-                },
-                {
-                    "@id": "reproschema:Field"
-                }
-            ]
+            "schema:rangeIncludes": {
+                "@id": "reproschema:addProperties"
```

Comment:
@satra is this correct? both `@id` of this property and `rangeIncludes` point to `reproschema:addProperties` .. I am a lil' confused if this is correct..? or should i name its schema something else and point `rangeIncludes` to this new name?

Reply:
that should not be the case ever. generally some id cannot point to itself as its range. as an example see this: https://schema.org/text.jsonld

in schema.org camelcase is used for properties while classes are capitalized.

Reply:
thanks for the example!